java/build.gradle.kts

```diff
@@ -19,14 +19,28 @@ java {
 
 spotless {
     java {
-        googleJavaFormat("1.9")
+        googleJavaFormat("1.15.0")
     }
 }
 
+val javaagentDependency by configurations.creating {
+    extendsFrom()
+}
+
 dependencies {
     compileOnly(platform("io.opentelemetry:opentelemetry-bom:1.25.0"))
     compileOnly(platform("io.opentelemetry:opentelemetry-bom-alpha:1.25.0-alpha"))
     // Already included in wrapper so compileOnly
     compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure-spi")
     compileOnly("io.opentelemetry:opentelemetry-sdk-extension-aws")
+    javaagentDependency("software.amazon.opentelemetry:aws-opentelemetry-agent:1.26.0")
```

Comment:
Can we keep the version at `1.25.0` for now to match with other deps as they will be updated as a whole later. It wouldn't really matter though.

Reply:
There is no version 1.25.0 :)